components/dialog/test/dialog-ifrau-contents.vdiff.html

```diff
@@ -18,24 +18,13 @@
 <body class="d2l-typography">
 	<d2l-button id="open">Open Dialog</d2l-button>
 
-	<d2l-dialog id="ifrau-dialog" title-text="Title" width="400">
+	<d2l-dialog id="ifrau-dialog" title-text="Title" width="400" opened>
 		<div>Line 1</div>
 		<div>Line 2</div>
 		<div>Line 3</div>
 		<div>Line 4</div>
 		<d2l-button slot="footer" primary>Yes</d2l-button>
 		<d2l-button slot="footer" id="cancel">No</d2l-button>
 	</d2l-dialog>
-
-	<script>
-		const dialog = document.getElementById('ifrau-dialog');
-		dialog.addEventListener('d2l-dialog-open', () => {
-			window.parent.postMessage('ifrau-dialog-open', '*');
```

Comment:
Are these postMessage values no longer being used?

Reply:
Yeah, they were used to open the dialog (and then close it in `reset` for the next test).  Now that the tests are isolated I can just start with the dialog opened.  But if we _want_ to test this functionality, I could add it back, or maybe just a dedicated test for opening and one for closing?